base/strings/utf8proc.jl

```diff
@@ -28,6 +28,15 @@ isvalid(value)
 Returns `true` if the given value is valid for that type. Types currently can
 be either `Char` or `String`. Values for `Char` can be of type `Char` or [`UInt32`](@ref).
 Values for `String` can be of that type, or `Vector{UInt8}`.
+
+# Examples
+```jldoctest
+julia> isvalid(Char(0xd800))
+false
+
+julia> isvalid(Char(0xd799))
+true
+```
```

Comment:
Actually, should these examples be on the method just above?

Reply:
Corrected